### ISSUE TITLE
How deploy etherpad on heroku?

### ISSUE THREAD

**Issue by yorbicardozo24** [ether/etherpad-lite#3630]
I looked for a lot of documents to do it but I couldn't, it took a long time and I haven't managed to do it.

Someone to help me with a step by step to achieve it

**Comment by lpagliari** [comment]
@yorbicardozo24 I've never done it, but here's what I've found: there's a [wiki page](https://github.com/ether/etherpad-lite/wiki/Deploying-etherpad-lite-on-Heroku) describing how to do it, although I don't know how up-to-date it is (it's from 2014). The link there is broken, but [this repo](https://github.com/citizenos/etherpad-lite-heroku) seems to be the one that is mentioned.

One other source I've found by searching "etherpad heroku" on google was [this one](https://elements.heroku.com/buttons/waterloo/etherpad-lite-heroku), but I don't know if it is correct or not.

Hope this helps.
